src/response_type.rs

```diff
@@ -0,0 +1,92 @@
+use ffi;
+use glib::translate::{FromGlib, ToGlib, ToGlibPtr, ToGlibPtrMut, from_glib};
+use glib::value::{FromValue, FromValueOptional, SetValue};
+use glib::{StaticType, Type, Value};
+use gobject_ffi;
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Ord, PartialOrd, Hash)]
+pub enum ResponseType {
+    None,
+    Reject,
+    Accept,
+    DeleteEvent,
+    Ok,
+    Cancel,
+    Close,
+    Yes,
+    No,
+    Apply,
+    Help,
+    Other(u16),
+    #[doc(hidden)]
+    Unknown(i32),
```

Comment:
Why did it lose its two underscores in the beginning?

Reply:
Want me to put them back? At first I just replaced it with `Other` and then put it back.

Reply:
I'd prefer that with underscores, yes